Project.toml

```diff
@@ -16,16 +17,18 @@ URIs = "5c2747f8-b7ea-4ff2-ba2e-563bfd36b1d4"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
-Base64 = "1"
-Dates = "1"
+Base64 = "<0.0.1, 1"
+Dates = "<0.0.1, 1"
 HTTP = "0.9.8, 1"
 JSON3 = "1.5.1"
-Mmap = "1"
+Logging = "<0.0.1, 1"
+Mmap = "<0.0.1, 1"
 SaferIntegers = "2.5.1, 3"
 StructTypes = "1.2.3"
+Test = "<0.0.1, 1"
```

Comment:
Not needed, I think?
```suggestion
```

Reply:
Currently, there is no real need to have compat entries for stdlibs that are test-dependencies. For example, AutoMerge will not complain about it.

However, the proposal for the future of stdlibs is that all (or almost all) stdlibs will become "upgradeable". This means that in the future, it will become possible for us to release a breaking 2.0 release of the Test.jl stdlib. So, in order to future-proof ourselves against those future breaking releases, we need to start adding compat entries for stdlibs that are test-dependencies.